README.md

```diff
@@ -1,187 +1,130 @@
-# MailPoet.
+# MailPoet
 
 MailPoet done the right way.
 
-# Install.
-
-- Install system dependencies:
-```
-php
-nodejs
-wordpress
-```
-
-- Clone the repo in `wp-content/plugins`.
-
-- Install composer.
-```sh
+# Contents
+
+- [Setup](#setup)
+- [Frameworks and libraries](#frameworks-and-libraries)
+- [Files structure](#files-structure)
+- [Workflow Commands](#workflow-commands)
+
+# Setup
+
+## Requirements
+- PHP 5.6+
+- NodeJS
+- WordPress
+- Docker & Docker Compose
+
+## Installation
+```bash
+# go to WP plugins directory
+$ cd path_to_wp_directory/wp-content/plugins
+# clone this repository
+$ git clone https://github.com/mailpoet/mailpoet.git
+$ cd mailpoet
+# create the .env file
+$ cp .env.sample .env
+# change the values on .env file
```

Comment:
This step might be confusing. We require a lot of api keys in .env file but I think that some of them are not critical to be able to do some development. e.g. if developer doesn't have aws credentials the AWS SES unit test will fail.
Not everyone has an AWS account or Mailchimp account and I can't imagine that someone will create all these accounts (AWS requires also some configuration) during this step.
I suggest to divide .env.sample file by a comment line to required and optional settings and point out that without the optional settings some functions will not work and some tests will fail.
What do you think?

Reply:
Yes, I agree. That's what I was discussing here https://mailpoet.slack.com/archives/G0A1L2KHV/p1539079892000100

I will edit `.env.sample` to make it clear.